errtracker/errtracker.go

```diff
@@ -160,6 +168,104 @@ func detectVirt() string {
 	return strings.TrimSpace(string(output))
 }
 
+func journalError() string {
+	// TODO: look into using systemd package (needs refactor)
+	output, err := exec.Command("journalctl", "--utc", "--boot", "--priority=warning..err", "--lines=1000").CombinedOutput()
+	if err != nil {
+		if len(output) == 0 {
+			return fmt.Sprintf("error: %v", err)
+		}
+		output = append(output, fmt.Sprintf("\nerror: %v", err)...)
+	}
+	return string(output)
+}
+
+func procCpuinfoMinimal() string {
+	buf, err := ioutil.ReadFile(procCpuinfo)
+	if err != nil {
+		// if we can't read cpuinfo, we want to know _why_
+		return fmt.Sprintf("error: %v", err)
+	}
+	idx := bytes.LastIndex(buf, []byte("\nprocessor\t:"))
```

Comment:
Can you add a comment explaining the rationale here? What is it that we are trying to collect?\
This will only work on x86. Perhaps we want to use `lscpu` instead?